staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/secretbox/secretbox_test.go

```diff
@@ -77,12 +77,35 @@ func TestSecretboxKeyRotation(t *testing.T) {
 	}
 }
 
-func BenchmarkSecretboxRead_32_1024(b *testing.B)        { benchmarkSecretboxRead(b, 32, 1024, false) }
-func BenchmarkSecretboxRead_32_16384(b *testing.B)       { benchmarkSecretboxRead(b, 32, 16384, false) }
-func BenchmarkSecretboxRead_32_16384_Stale(b *testing.B) { benchmarkSecretboxRead(b, 32, 16384, true) }
+func BenchmarkSecretboxRead(b *testing.B) {
+	tests := []struct {
+		keyLength   int
+		valueLength int
+		expectStale bool
+	}{
+		{keyLength: 32, valueLength: 1024, expectStale: false},
+		{keyLength: 32, valueLength: 16384, expectStale: false},
+		{keyLength: 32, valueLength: 16384, expectStale: true},
+	}
+	for _, t := range tests {
+		name := fmt.Sprintf("%vKeyLength/%vValueLength/%vExpectStale", t.keyLength, t.valueLength, t.expectStale)
+		b.Run(name, func(b *testing.B) { benchmarkSecretboxRead(b, t.keyLength, t.valueLength, t.expectStale) })
```

Comment:
could you move the function body onto its own line?

Reply:
OK